R/render.R

```diff
@@ -329,11 +331,19 @@ use_lang_chapter <- function(chapters_list, language_code,
 
 add_links <- function(path, main_language, # nolint: cyclocomp_linter
                       language_code, site_url, type, config, output_folder,
-                      path_language) {
+                      path_language, project_dir) {
   html <- xml2::read_html(path)
 
   document_path <- path
 
+  lang_profile <- file.path(
+    project_dir, paste0("_quarto-", language_code, ".yml")
+  )
+  if (fs::file_exists(lang_profile)) {
+    lang_config <- read_yaml(lang_profile)
+    config <- utils::modifyList(config, lang_config)
```

Comment:
so happy it was that simple! https://masalmon.eu/2023/06/06/basic-patterns/#combine-a-list-of-default-values-with-a-list-of-custom-values